### DISCUSSION TITLE
What famous rap artist studied ballet?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#421] [Galaxy Trivia] [answered]
**Hint:** Changes ✌️

**Comment by siddarthkay** [comment, marked as answer]
Tupac Shakur!

**Reply by ParanoidUser** [reply]
Well done! 👏👏 Congratulations on your well deserved [Galaxy Brain](https://github.com/siddarthkay?tab=achievements&achievement=galaxy-brain) badge. Continue to unlock the next badge levels (Bronze for 8 answers), and don't hesitate to post your own questions to the community. 😊🎉

**Comment by peterbaccaro** [comment]
In 1986, 2Pac's family moved to Baltimore and **Tupac Shakur** (aka **2Pac**) was enrolled at the Baltimore School for the Arts in order to study ballet and acting.
